rpcserver.go

```diff
@@ -3290,8 +3290,11 @@ func (r *rpcServer) WalletBalance(ctx context.Context,
 		// Get confirmed balance, from txs that have >= 1 confirmations.
 		// TODO(halseth): get both unconfirmed and confirmed balance in
 		// one call, as this is racy.
+		if in.MinConfs == 0 {
```

Comment:
If we pass in a negative value that might cause an error in `ConfirmedBalance`. Should we do `if in.MinConfs <= 0`?

Reply:
Makes total sense. Done 🏁